CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 ### Fixed
 
+- Some issues with the structured data on the website https://github.com/tuist/tuist/pull/975 by @pepibumur.
```

Comment:
Was thinking about the changelog - maybe it's worth moving what we change on the website in a separate section? I guess people updating Tuist might not be so interested about SEO changes. We later move all items to the GitHub release page and it starts to make even less sense...

Reply:
I agree this information is not relevant to the users. I think it's fine if we don't reflect it in the CHANGELOG because that's in the git history anyways.